google/cloud/forseti/scanner/scanner_builder.py

```diff
@@ -51,60 +55,79 @@ def build(self):
             list: Scanner instances that will be run.
         """
         runnable_scanners = []
-        for scanner in self.scanner_configs.get('scanners'):
-            if scanner.get('enabled'):
-                module_path = 'google.cloud.forseti.scanner.scanners.{}'
-
-                requirements_map = scanner_requirements_map.REQUIREMENTS_MAP
-                if not scanner.get('name') in requirements_map:
-                    log_message = (
-                        'Configured scanner is undefined '
-                        'in scanner requirements map : %s')
-                    LOGGER.error(log_message, scanner.get('name'))
-                    continue
-
-                module_name = module_path.format(
-                    scanner_requirements_map.REQUIREMENTS_MAP
-                    .get(scanner.get('name'))
-                    .get('module_name'))
-
-                try:
-                    module = importlib.import_module(module_name)
-                except (ImportError, TypeError, ValueError):
-                    LOGGER.exception('Unable to import %s\n', module_name)
-                    continue
-
-                class_name = (
-                    scanner_requirements_map.REQUIREMENTS_MAP
-                    .get(scanner.get('name'))
-                    .get('class_name'))
-                try:
-                    scanner_class = getattr(module, class_name)
-                except AttributeError:
-                    LOGGER.exception('Unable to instantiate %s', class_name)
-                    continue
-
-                # Simple way to find the path to folders directory no matter
-                # where forseti runs.
-                rules_path = self.scanner_configs.get('rules_path')
-                if rules_path is None:
-                    scanner_path = inspect.getfile(scanner_class)
-                    rules_path = scanner_path.split('/google/cloud/forseti')[0]
-                    rules_path += '/rules'
-
-                rules_filename = (scanner_requirements_map.REQUIREMENTS_MAP
-                                  .get(scanner.get('name'))
-                                  .get('rules_filename'))
-                rules = '{}/{}'.format(rules_path, rules_filename)
-                LOGGER.info('Initializing the rules engine:\nUsing rules: %s',
-                            rules)
-
-                scanner = scanner_class(self.global_configs,
-                                        self.scanner_configs,
-                                        self.service_config,
-                                        self.model_name,
-                                        self.snapshot_timestamp,
-                                        rules)
-                runnable_scanners.append(scanner)
+        if self.scanner_name == 'external_project_access_scanner':
+            scanner = self.make_scanner(external_project_access_scanner.
```

Comment:
How about we call this `instantiate_scanner()`?
```
scanner = self.instantiate_scanner(
    external_project_access_scanner.ExternalProjectAccessScanner,
    'external_project_access_rules.yaml')
```

Reply:
modified as suggested